static/js/components/SourceTable.jsx

```diff
@@ -666,6 +666,24 @@ const SourceTable = ({
     );
   };
 
+  const getSavedBy = (source) => {
+    // Get the user who saved the source to the specified group
+    if (groupID !== undefined) {
+      const group = source.groups.find((g) => g.id === groupID);
+      return group?.saved_by.username;
+    }
+    // Otherwise, get whoever saved it last
```

Comment:
When would we ever run into this?

Reply:
I think this happens when loading the general sources page, rather than the group sources page. They both use `SourceTable`.

Reply:
Right yeah, the `groupID` here is the param passed into the `SourceTable` based on whether it's on the group sources page and thus has a specific group associated for the data displayed, or the general sources page

Reply:
Ah yeah, I forgot this is used for both GroupSources and general Sources pages